README.rst

```diff
@@ -169,22 +169,22 @@ their coordinates. See below for further details:
 
 .. code-block:: text
 
-    GET /api/v1/device/{id}/location/
-    PUT /api/v1/device/{id}/location/
+    GET /api/v1/controller/device/{id}/location/
+    PUT /api/v1/controller/device/{id}/location/
 
 If a location has multiple devices, then the list of all such devices and related
 information can be accessed via the following endpoint:
 
 .. code-block:: text
 
-    GET /api/v1/location/{pk}/device/
+    GET /api/v1/controller/location/{pk}/device/
 
 To access all the locations that have devices deployed in them, use the following
 endpoint, which provides a paginated list of such locations in GeoJSON format:
 
 .. code-block:: text
 
-    GET /api/v1/location/geojson/
+    GET /api/v1/controller/location/geojson/
```

Comment:
I forgot about this. Can you please move these endpoints to the section dedicated to the REST API?

In this section we can then simply refer to the endpoints by linking the first of them.

Reply:
Done. Also here the endpoints for sending commands to devices are wrong:- https://github.com/openwisp/openwisp-controller#list-commands-of-a-device

Shouldn't there be `controller` in place of `connection`.